src/pages/Article/Article.js

```diff
@@ -1,30 +1,225 @@
 
 const Article = () => {
+  //article 내용 가져오기
+  const [getArticle, setGetArticle] = useState({});
+  const params = useParams();
+  const writingsId = params.id;
+
+  // `http://10.58.52.136:3000/writings/:writing_id`
+  useEffect(() => {
+    fetch('http://10.58.52.136:3000/writings/11')
+      .then(response => response.json())
+      .then(result => setGetArticle(result.writing[0]));
+  }, [writingsId]);
+
+  const {
+    title,
+    writer,
+    content,
+    subscribers,
+    author_id,
+    likes,
+    article_id,
+    price,
+    header_image,
+    color,
+  } = getArticle;
+
+  //추천글 mockdata 내용 가져오기
+  const [getRecommend, setGetRecommend] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/recommend.json')
+      .then(response => response.json())
+      .then(result => setGetRecommend(result));
+  }, []);
+
+  //작가 구독하기
+  const [clickCount, setClickCount] = useState(0);
+  const AddSubscribe = () => {
+    setClickCount(clickCount + 1);
+    let copy = { ...getArticle };
+    if (clickCount % 2 === 0) {
+      copy.subscribers = Number(subscribers) + 1;
+      setGetArticle(copy);
+    } else {
+      copy.subscribers = Number(subscribers) - 1;
+      setGetArticle(copy);
+    }
+  };
+  //구독하기 정보 BE로 전달
+  const Subscribe = useEffect(() => {
+    fetch(`http://10.58.52.92:3000/likes/authors/${author_id}`, {
+      method: 'POST',
+      headers: {
+        'content-Type': 'application/json;charset=utf-8',
+        // Authorization: localStorage.getItem('accessToken'),
+      },
+      body: JSON.stringify({
+        id: 1,
+      }),
+    }).then(response => response.json());
+  }, [clickCount, author_id]);
+
+  //글 좋아요
+  const [likeCount, setLikeCount] = useState(0);
+  const AddLikes = () => {
+    setLikeCount(likeCount + 1);
+    let copy = { ...getArticle };
+    if (likeCount % 2 === 0) {
+      copy.likes = Number(likes) + 1;
+      setGetArticle(copy);
+    } else {
+      copy.likes = Number(likes) - 1;
+      setGetArticle(copy);
+    }
+  };
+
+  //글 좋아요 정보 BE로 전달
+  const LikeArticle = useEffect(() => {
+    fetch(`http://10.58.52.92:3000/likes/writing/${article_id}`, {
+      method: 'POST',
+      headers: {
+        'content-Type': 'application/json;charset=utf-8',
+        // Authorization: localStorage.getItem('accessToken'),
+      },
+      body: JSON.stringify({
+        id: 1,
+      }),
+    }).then(response => response.json());
+  }, [likeCount, article_id]);
+
+  const style = {
+    backgroundColor: color,
+  };
+
   return (
     <>
       <Header>
-        <Title>제목 고양이</Title> bnbn
+        {color ? <Color style={style} /> : <BackImg src={header_image} />}
+        <Title>{title}</Title>
         <Writer>
-          by 책공장
-          <Icon className="material-symbols-outlined">favorite</Icon>
+          by {writer}
+          {likeCount % 2 === 0 ? (
+            <Icon
+              className="material-symbols-outlined"
+              onClick={(LikeArticle, AddLikes)}
```

Comment:
올바르게 동작하지 않을 것 같네요!
이 부분도 위 리뷰와 관련해서 내일 함께 보시죠!
```suggestion
              onClick={() => {
                likeArticle();
                addLikes();
              }}
```